packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -183,6 +183,9 @@ export class SceneTree {
   @Prop({ mutable: true })
   public controller?: SceneTreeController;
 
+  @Prop({ mutable: true })
+  public multiSelectedRows?: Row[];
```

Comment:
This prop will need to be kept up-to-date when selections happen outside the tree. If not needed, maybe we exclude it for now?

Reply:
Connect-app watches this variable to see get all the items that were included in the multi-selection: https://github.com/Vertexvis/connect-app/pull/265/files#diff-73d237b734a2c7c0749a9d783e73634e1fdba135954dcb73ad4511eb7902cb88R128. 
Connect-app doesn't know the index of the scene tree items, so it wasn't able to get this information on its own, and without it, _show/hide only selected_ was only changing the state of the scene tree items that were actually clicked on instead of the entire range.

Reply:
I think there's a few issues here:

* The `rows` property that you're filtering on doesn't include the whole list of items. The tree will lazy-load items and purge items from this list to keep memory pressure low. The result is that users may see gaps in there selection if they perform a range selection over multiple pages of data.
* There could be some large data sets that we're sending over the wire when performing scene item overrides. I think we may need some additional help on the backend here to resolve this. IIRC, @jareddellitt had some thoughts here.
* By adding a property to the component, I'd expect this field to return all the rows that are currently selected. Selection can happen in other methods besides `performMultiSelection`, e.g. `selectItem` and through the viewer. If you don't want to maintain this bookkeeping, I'd recommend instead to return the rows from `selectRange`.